src/dialects/abstract/query-interface-typescript.ts

```diff
@@ -12,16 +14,99 @@ export class AbstractQueryInterfaceTypeScript {
   }
 
   /**
-   * Creates a new database schema.
+   * Create a new database schema.
    *
    * **Note:** this is a schema in the [postgres sense of the word](http://www.postgresql.org/docs/9.1/static/ddl-schemas.html),
-   * not a database table. In mysql and sqlite, this command will do nothing.
+   * not a database table. In mysql and mariadb, this command will instead create a database.
    *
-   * @param schema
-   * @param options
+   * @see
+   * {@link Model.schema}
+   *
+   * @param schema Name of the schema
+   * @param [options={}]
+   * @param [options.collate=null]
+   * @param [options.charset=null]
+    *
+   * @returns
```

Comment:
You don't need to document the individual options here, that should be documented in CreateSchemaOptions

There is also no need to indicate that the parameter is optional, that's already part of the TypeScript definition (no `[]` around the name, nor `=null` or `={}`)

Lastly, `@returns` doesn't document anything, and can be removed

The same comments apply for most other doc blocks in this PR